app/controllers/work_packages/activities_tab_controller.rb

```diff
@@ -43,7 +43,8 @@ def index
       WorkPackages::ActivitiesTab::IndexComponent.new(
         work_package: @work_package,
         filter: @filter,
-        last_server_timestamp: get_current_server_timestamp
+        last_server_timestamp: get_current_server_timestamp,
+        deferred: params[:deferred] == "true"
```

Comment:
```suggestion
        deferred: ActiveRecord::Type::Boolean.new.cast(params[:deferred])
```

More complete & Safer

```ruby
[8] pry(main)> ActiveRecord::Type::Boolean.new.cast('true')
=> true
[9] pry(main)> ActiveRecord::Type::Boolean.new.cast('1')
=> true
[10] pry(main)> ActiveRecord::Type::Boolean.new.cast('0')
=> false
[11] pry(main)> ActiveRecord::Type::Boolean.new.cast(nil)
=> nil
```